src/components/feedback-message/FeedbackMessage.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { WarningIcon, InfoIcon } from '../icon';
+import { Tooltip, TooltipTrigger } from '../tooltip';
+import styles from './FeedbackMessage.css';
+
+const FeedbackMessage = ({ children, type, textColor, iconPosition, variant, tooltipChildren, className }) => {
+    const finalClassNames = classNames(
+        styles.feedbackMessage,
+        styles[type],
+        styles[iconPosition],
+        styles[variant],
+        className
+    );
+
+    const renderTooltip = () => {
+        if (!type) {
+            return;
+        }
+
+        return tooltipChildren ?
+            <TooltipTrigger tooltip={ <Tooltip placement="bottom" style={ { maxWidth: 286 } }>{ tooltipChildren }</Tooltip> }>
+                { <div className={ styles.iconWrapper }>{ renderIcon() }</div> }
+            </TooltipTrigger> :
+            renderIcon();
+    };
+
+    const renderIcon = () => type === 'error' ?
```

Comment:
⚠️Nitpicking Alert 😅.
As I mentioned above, I think that multi line ternary conditions are difficult to read.

Something like:
```js
const renderIcon = () => {
    if (type === 'error') {
        return <WarningIcon className={ styles.icon } />;
    }

    return <InfoIcon className={ styles.icon } />;
};
```

This also enables a way to add another icon type in the future with less effort. Again, if you prefer it, leave it this way 😁